### ISSUE TITLE
[Google Cloud Platform Search] Add Support for multiple Google Accounts

### ISSUE THREAD

**Issue by DominiqueLade** [raycast/extensions#4644]
### Extension

https://www.raycast.com/klaa97/gcp-search

### Description

Currently, I am logged in into two Google accounts at the same time. A private and a company GCP account. The private account is the first in the list of logged-in accounts. The GCP resources are in the second account.

Using the "Search Project" of the "GCP Search" opens the link automatically in the web browser with the first account. However, the GCP resources are linked to the second account. Therefore, the link is invalid and causes a permission error.

I would like to see an option to select the desired account within Raycast. It appears that it is enough to add `?authuser=n` to the link, while `n` is the `n`th account the user is logged in. In my case, it would be `?authuser=1`.

I would like to set the `n` authuser as a settings parameter within the Raycast extension.

### Who will benefit from this feature?

Everyone who uses several Google accounts and the Google account with the GCP account of interest is not the main GCP account.

For example, having a private and a company GCP account already causes an issue.

### Anything else?

See: https://superuser.com/questions/1686288/open-google-cloud-console-with-a-certain-user-account

**Comment by DominiqueLade** [comment]
@klaa97 Thanks for taking the time to response. 

I do agree that the solution I proposed is not perfect. It is merely a workaround since it is, to my knowledge, not possible to directly generate a link to the Google Cloud console with the correct user / account specified.

However, changing settings in the `gcloud cli`will not solve the problem. In Raycast I do see the right projects. However, once I select a project and a product, the link will open in my default web browser. Since I am logged in, in the web browser with several Google accounts, the link will always use the first Google account. Attaching `?authuser=1`will at least allow the user to specify in which user account the link should be opened.

I believe the links to the products are hardcoded here: https://github.com/raycast/extensions/blob/main/extensions/google-cloud-platform-search/src/consoleProducts.tsx. Exposing the `authuser`as a configuration option within Raycast and then changing the links to something like `https://console.cloud.google.com/iam-admin?project=${projectId}?authuser=${authuser}` should do the trick.

Unfortunately, I do not have the capacity to contribute. In case someone has a similar issue, I tried to build a workaround with a browser selector like Velja.
